### PR TITLE
Alter snippet code

### PR DESCRIPTION
**Pull request by Guntrisoft**
- Alter snippet code so that the markup which goes into the
editor is valid XHTML markup (HR tag)
- changed snippet example in admin accordingly

### CODE REVIEW

**Comment by Guntrisoft** [comment]
@benbarnett @jongilbraith

**Comment by benbarnett** [comment]
Can we pull the snippet code from a centralised source? Just a partial or something?

File: lib/publify_textfilter_ma_says.rb [hunk 1]
```diff
@@ -6,22 +6,22 @@ class MaSays< TextFilterPlugin::MacroPost
...
+        url = attrib['data-snippet-url']
+        text = attrib['data-snippet-text']
 
         "<div class=\"snippet next-steps\">
            <div class=\"next-steps__heading\">
              <span>MA</span> says
            </div>
 
-           <a data-ck href=\"#{url}\" class=\"next-steps__container-link\">
+           <a href=\"#{url}\" class=\"next-steps__container-link\">
```
Comment (benbarnett) [inline lib/publify_textfilter_ma_says.rb]: Is this still right? Or should it be the `hr` now

Comment (Guntrisoft) [inline lib/publify_textfilter_ma_says.rb]: As discussed this is the code that is actually output to the article page, instead of the <hr />

**Comment by jongilbraith** [comment]
:+1: 

@benbarnett would have to look into moving it into a partial, but should be possible without too much trouble.

**Comment by benbarnett** [comment]
Cool. Appreciate there are other features with higher priority. As @Guntrisoft said, perhaps if/when a 3rd snippet comes along, we can look at improving this and reducing some of the duplication.

:+1:

**Pull request merged**